charts/ceph-csi-cephfs/templates/storageclass.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.storageClass.create -}}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: {{ .Values.storageClass.name }}
+   namespace: {{ .Release.Namespace }}
+   labels:
+     app: {{ include "ceph-csi-cephfs.name" . }}
+     chart: {{ include "ceph-csi-cephfs.chart" . }}
+     release: {{ .Release.Name }}
+     heritage: {{ .Release.Service }}
+provisioner: {{ .Values.driverName }}
+parameters:
+  clusterID: {{ .Values.storageClass.clusterID }}
+  fsName: {{ .Values.storageClass.fsName }}
+{{- if .Values.storageClass.pool }}
+  pool: {{ .Values.storageClass.pool }}
+{{- end }}
+{{- if .Values.storageClass.fuseMountOptions }}
+  fuseMountOptions: "{{ .Values.storageClass.fuseMountOptions }}"
+{{- end }}
+{{- if .Values.storageClass.kernelMountOptions }}
+  kernelMountOptions: "{{ .Values.storageClass.kernelMountOptions }}"
+{{- end }}
+{{- if .Values.storageClass.mounter }}
+  mounter: "{{ .Values.storageClass.mounter }}"
+{{- end }}
+{{- if .Values.storageClass.volumeNamePrefix }}
+  volumeNamePrefix: "{{ .Values.storageClass.volumeNamePrefix }}"
+{{- end }}
+  csi.storage.k8s.io/provisioner-secret-name: {{ .Values.storageClass.provisionerSecret }}
+  csi.storage.k8s.io/provisioner-secret-namespace: {{ .Release.Namespace }}
+  csi.storage.k8s.io/controller-expand-secret-name: {{ .Values.storageClass.controllerExpandSecret }}
+  csi.storage.k8s.io/controller-expand-secret-namespace: {{ .Release.Namespace }}
+  csi.storage.k8s.io/node-stage-secret-name: {{ .Values.storageClass.nodeStageSecret }}
+  csi.storage.k8s.io/node-stage-secret-namespace: {{ .Release.Namespace }}
+reclaimPolicy: {{ .Values.storageClass.reclaimPolicy }}
+allowVolumeExpansion: {{ .Values.storageClass.allowVolumeExpansion }}
+{{- if .Values.storageClass.mountOptions }}
+  mountOptions:
+    {{- range .Values.storageClass.mountOptions }}
+    - {{ . }}
+    {{- end }}
+  {{- end -}}
+{{- end }}
```

Comment:
```suggestion
{{- if .Values.storageClass.mountOptions }}
mountOptions:
  {{- range .Values.storageClass.mountOptions }}
  - {{ . }}
  {{- end }}
{{- end }}
{{- end -}}
```

Reply:
Without the suggested changes Helm complains that 'mapping values are not allowed in this context'

Reply:
> Without the suggested changes Helm complains that 'mapping values are not allowed in this context'

Thanks for catching it! I fixed and re-verified the changes now. Can you please verify if everything works fine for you now?

Reply:
The changes work as indended for me!

Reply:
Thanks for confirming :)